app/models/proposal.rb

```diff
@@ -74,14 +72,6 @@ def root_step
     steps.where(parent: nil).first
   end
 
-  def parallel?
-    flow == "parallel"
-  end
-
-  def linear?
-    flow == "linear"
-  end
-
```

Comment:
`parallel?` and `linear?` still seem like useful methods to have? Maybe they should just look at the root step instead?